mne/minimum_norm/spatial_resolution.py

```diff
@@ -332,3 +327,8 @@ def _rectify_resolution_matrix(resmat):
                     (shape[0], shape[1], resmat.shape[0], resmat.shape[1]))
 
     return resmat
+
+###############################################################################
+# References
+# ----------
+# .. footbibliography::
```

Comment:
why is this necessary?

Reply:
agree, this is not needed.  in a *code* file the references section goes within the docstring where the reference is cited.  In a *tutorial* file you want something like this just once at the very end.

Reply:
That was a remnant, so we'll get that out.